apps/condo/domains/billing/schema/BillingIntegration.js

```diff
@@ -98,6 +98,7 @@ const BillingIntegration = new GQLListSchema('BillingIntegration', {
             type: Relationship,
             ref: 'BillingIntegrationAccessRight.integration',
             many: true,
+            access: { create: false, update: false },
```

Comment:
Probably, we also need to lint that you define all possible operations: create, read, update (optional)

Reply:
It seems that if you do not define any access, it auto sets to true. I added a check just in case ...

Reply:
explicit is better than implicit